public_html/wp-content/mu-plugins/blocks/webpack.config.js

```diff
@@ -17,6 +17,8 @@ const webpackConfig = {
 		minimize: true,
 	},
 
+	devtool: 'production' === NODE_ENV ? 'none' : 'cheap-module-eval-source-map',
```

Comment:
@vedanshujain , @coreymckrill : FYI.

This adds about `500ms` to the `watch` process, but seems worth it. [The only faster options](https://webpack.js.org/configuration/devtool/) would result in more difficult debugging experiences, but we can experience with them if the slower rebuilds have negative impacts on our workflow.

Reply:
Ideally we should not be minimizing files on local, instead of using source map.

Reply:
🤔 

I love the idea of not needing to add the extra time to the rebuild process, but I'm not sure if debugging would be too hard. Currently the code gets minified, but at least in FireFox it shows it formatted, so that isn't a problem for me. The bigger problem is that the code you see in the browser is the _generated_ code, rather than the original _source_, which has a few problems:

1) It's full of verbose/obscure var names, polyfills, transpiled alternatives, etc, so it's difficult to read and understand
2) It's inconvenient to manually trace line in the generated source back to the line in the original source.

There are advantages to that, though, in the case where a bug _results from_ the transpilation, but that's probably fairly rare.

I could go either way. @vedanshujain , @coreymckrill , what has your debugging experience been like so far? Have you found it difficult to track down errors? If not, then the source maps may be unnecessary. They seemed helpful to me, but that could just be because I'm newer to this.

If we do remove them, then maybe an alternative idea would be to add a 3rd build mode for debugging, and in that mode the source maps would be generated. e.g,

```diff
"scripts": {
	"start": "cross-env BABEL_ENV=development NODE_ENV=development webpack --watch",
	"build": "cross-env BABEL_ENV=production NODE_ENV=production webpack",
+	"debug": "cross-env BABEL_ENV=development NODE_ENV=debug webpack",
	"lint:js": "wp-scripts lint-js ."
}
```

Reply:
With FF Developer's script debug console, I've usually been able to decipher the generated code and figure out which line of the source is causing the bug. I'm not opposed to source maps though. I just never took the time to figure out how to set them up and get them to work right...

Reply:
There have been some instances where instead of where the error message came from, I have been presented with a link which make me go to a webpage to see the error message.
Other than this though debugging experience has been good.

Reply:
hm, ok, I'll switch it to add the `debug` mode instead, then, so that we can get good source maps when we need them, but that it won't slow down the regular rebuild process.

Reply:
I would keep that as a low priority though. This does not cause much pain.

Reply:
Done in 508984cce4e5a65bc65f907ad369ee7e64928f9c. Run `npm run debug` to enable the source maps.